### PR TITLE
Docs/bunit

### PR DESCRIPTION
**Pull request by lmontopo**

### CODE REVIEW

File: source/includes/tasks.md [hunk 1]
```diff
@@ -161,7 +161,7 @@ See the [Include Parameters](#include-parameters) section for more details.
...
 Parameter | Description
 ----------|---------------------
-tags      | includes a list of tags associated to the task
-related   | includes a list of related tasks
-problem   | includes the problem that the task is related to
-how_tos   | includes a list of how-tos
+tags      | Includes a list of tags associated to the task
+related   | Includes a list of related tasks
+problem   | Includes the problem that the task is related to
+how_tos   | Includes a list of how-tos
```
Comment (funkaoshi) [inline source/includes/tasks.md]: `Includes a list of applicable how-tos for this task` might be more descriptive.

File: source/includes/tasks.md [hunk 2]
```diff
@@ -96,8 +96,8 @@ See the [Expand Parameters](#expand-parameters) section for more details.
 
 Parameter   | Description
 ------------|---------------------
-status      | expands the status field associated with the task
-description | expands the description field associated with the task
+status      | Expands the status field associated with the task
```
Comment (funkaoshi) [inline source/includes/tasks.md]: Maybe a larger discussion: this seems redundant. We know that we are going to expand the status and the description. Perhaps we should explain what new fields will show up, or what this means. I think expand description splits out the amendments into their own field, for example.

Comment (lmontopo) [inline source/includes/tasks.md]: That sounds good.  Simon and I struggled a bit with the description for expands and includes parameters.  I will make some changes and then push them up for further review.

File: source/includes/tasks.md [hunk 3]
```diff
@@ -96,8 +96,8 @@ See the [Expand Parameters](#expand-parameters) section for more details.
 
 Parameter   | Description
 ------------|---------------------
-status      | expands the status field associated with the task
-description | expands the description field associated with the task
+status      | Status field returned with meaning, icon, name and slug sub-fields.
+description | Description field is returned with content and amendments sub-fields.
 
 ---
```
Comment (lmontopo) [inline source/includes/tasks.md]: @funkaoshi how do you feel about this?  Does the wording make sense? 
After I get feedback / after we agree on wording, I'll go and make similar changes all over.

Comment (funkaoshi) [inline source/includes/tasks.md]: Yeah I think that's better.

**Comment by funkaoshi** [comment]
:+1:

**Pull request merged**